lib/ui/widgets/member_tile.dart

```diff
@@ -24,30 +25,89 @@ class MemberTile extends StatelessWidget {
               fit: BoxFit.cover,
               fadeInDuration: const Duration(milliseconds: 200),
             ),
-            Container(
-              padding: const EdgeInsets.all(8),
+            const _BlackGradient(),
+            Align(
               alignment: Alignment.bottomLeft,
-              decoration: BoxDecoration(
-                color: Colors.black,
-                gradient: LinearGradient(
-                  begin: FractionalOffset.topCenter,
-                  end: FractionalOffset.bottomCenter,
-                  colors: [
-                    Colors.black.withOpacity(0.0),
-                    Colors.black.withOpacity(0.5),
-                  ],
-                  stops: const [0.4, 1.0],
+              child: Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: Text(
+                  member.displayName,
+                  style: Theme.of(context).primaryTextTheme.bodyText2,
                 ),
               ),
-              child: Text(
-                member.displayName,
-                style: Theme.of(context).primaryTextTheme.bodyText2,
+            ),
+            Positioned.fill(
+              child: Material(
+                color: Colors.transparent,
+                child: InkWell(onTap: openContainer),
               ),
-            )
+            ),
           ],
         );
       },
       openBuilder: (_, __) => ProfileScreen(pk: member.pk, member: member),
     );
   }
 }
+
+class _BlackGradient extends StatelessWidget {
+  static const _black00 = Color(0x00000000);
+  static const _black50 = Color(0x80000000);
+
+  const _BlackGradient();
+
+  @override
+  Widget build(BuildContext context) {
+    return const DecoratedBox(
+      decoration: BoxDecoration(
+        color: Colors.black,
+        gradient: LinearGradient(
+          begin: FractionalOffset.topCenter,
+          end: FractionalOffset.bottomCenter,
+          colors: [_black00, _black50],
+          stops: [0.4, 1.0],
+        ),
+      ),
+    );
+  }
+}
+
+class NameOnlyMemberTile extends StatelessWidget {
```

Comment:
I suggest renaming this class to 'DefaultMemberTile' as it will always have the default-avatar.jpg with the name. NameOnlyMemberTile might be confusing.

Reply:
Fair